### DISCUSSION TITLE
京东云百里，即jdcloud_re-cp-03这个设备编译后的固件实测

### DISCUSSION THREAD

**Discussion by zongheng23333** [immortalwrt/immortalwrt#1355] [Q&A]
京东云百里，即jdcloud_re-cp-03这个设备编译后的固件实测没有无线，
https://firmware-selector.immortalwrt.org/  这个网站上下载的也没有无线，应该就不是编译问题
![PixPin_2024-07-03_11-28-25](https://github.com/immortalwrt/immortalwrt/assets/29665054/08600537-7637-4673-9773-0c9d74624f2a)

**Comment by initzhu** [comment]
日志有吗？看你这主题是argon，你在构建时加入其他插件有哪些。是否刷入固件前/后恢复过出厂设置？

**Comment by 1715173329** [comment]
根据 #1357 的日志，初步判定 EEPROM 损坏。
输入命令 hexdump /dev/mmcblk0p2 看下校准数据是否正确。

**Reply by zongheng23333** [reply]
解决了 ，备份分区写了下    也不知道哪个分区有问题   反正把可疑分区都写了一下

**Reply by TomAThub** [reply]
> 解决了 ，备份分区写了下 也不知道哪个分区有问题 反正把可疑分区都写了一下

多次刷機後也碰到同樣狀況 但手上沒有備份 可否提供factory分區的備份供我試試能不能救回來？
